server/script.js

```diff
@@ -0,0 +1,11 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+
+export const options = {
+  vus: 300, // stimulate how many virtual users
```

Comment:
May want to see if this works as if I didnt do a ramp of VUs, I'd fail some requests and I think you can push up to 400 if you do this while holding to the requirements of RPS and error rate.